x-pack/plugins/apm/public/components/app/settings/agent_explorer/index.tsx

```diff
@@ -144,6 +155,7 @@ export function AgentExplorer() {
               start={start}
               end={end}
               dataTestSubj="agentExplorerServiceNameSelect"
+              fullWidth={true}
```

Comment:
Why `fullWidth` is needed here? I expect the `EuiFlexItem`will handle it

Reply:
I checked what you mentioned and for mobile views we get the desired effect without the `fullWidth`. Thanks